mage_ai/frontend/oracle/elements/Link/index.tsx

```diff
@@ -202,15 +203,16 @@ export const SHARED_LINK_STYLES = css<any>`
     text-overflow: ${props.textOverflow};
   `}
 
-  ${props => props.width && `
+  ${props => (props.minWidth || props.maxWidth) && `
```

Comment:
Are these props needed? `SHARED_TEXT_STYLES` already has minWidth and maxWidth props.

Reply:
And `LinkStyle` contains `SHARED_TEXT_STYLES`.

Reply:
The only difference is the existing props in `mage-ai/mage_ai/frontend/oracle/elements/Text/index.tsx` does not contain `display: block`, so maybe whatever Link component had the issue needs a `block` prop.

Reply:
oh I didn’t realize that it uses shared styles, the only thing is that Link needs `display: block` like you said

Reply:
anyways we can’t use `display: block` for `SuggestionsRow` bc it breaks the design of inline links, so it was more of a fix for the future. I’ll remove and only add `display: block`

Reply:
We have a `block` prop in `Link`, so check that.